Documentation/ExtensionArchitecture/Extbase/Reference/_FrontendPlugin/_tt_content.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+defined('TYPO3') or die();
+
+(static function (): void {
+    \TYPO3\CMS\Extbase\Utility\ExtensionUtility::registerPlugin(
+        // extension name, matching the PHP namespaces (but without the vendor)
```

Comment:
Extension name would be "BlogExample", the extension key would be "blog_example".

AFAIK is does not matter, you can use both. But the comment should be correct.